engine-cli/src/main/java/io/nosqlbench/engine/cli/NBCLIOptions.java

```diff
@@ -340,6 +350,9 @@ private LinkedList<String> parseGlobalOptions(final String[] args) {
                     showStackTraces = true;
                     arglist.removeFirst();
                     break;
+                case NBCLIOptions.ENABLE_DEDICATED_VERIFICATION_LOGGER:
+                    enableDedicatedVerificationLogger();
```

Comment:
There is a bug here. Args must be consumed explicitly on matching.

Reply:
Adding `arglist.removeFirst();`